modules/back-end/src/Api/appsettings.json

```diff
@@ -20,5 +20,9 @@
   "OLAP": {
     "ServiceHost": "http://da-server"
   },
-  "AllowedHosts": "*"
+  "Redis": {
+    "ConnectionString": "redis:6379"
+  },
+  "AllowedHosts": "*",
+  "LIGHT_VERSION": true
```

Comment:
"IS_PRO": false

Reply:
fixed in [070bb5e](https://github.com/featbit/featbit/pull/313/commits/070bb5ee9b1470bda4ec576142ff563b9563dce2)